tools/finish.nim

```diff
@@ -225,6 +225,8 @@ proc main() =
       if x.len == 0: continue
       let y = try: expandFilename(if x[0] == '"' and x[^1] == '"':
                                     substr(x, 1, x.len-2) else: x)
+              except OSError: 
+                if getCurrentExceptionMsg().endsWith("does not exist"): x else: ""
```

Comment:
Really hacky, how about:

```

except OSError as e:
  if e.errorCode == ?: x else: ""

```

Reply:
Thank you for you comment. Agree that it's hacky I've commit new one #12413 per your comment.